docs/en/compute/Spark-On-Kubernetes.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: global
+title: Deploy Alluxio on Kubernetes
+nickname: Kubernetes
+group: Deploying Alluxio
+priority: 3
+---
+
+Alluxio can be run on kubernetes. This guide demonstrates how to run a Spark job on Alluxio
+running in a kubernetes environment.
+
+* Table of Contents
+{:toc}
+
+## Basic Tutorial
+
+This tutorial walks through an example Spark job on Alluxio in kubernetes.
+
+### Prerequisites
+
+- A kubernetes cluster (version >= 1.8).
+
+### Download Binaries
+
+Download the desired Spark version.
+```bash
+wget http://apache.mirrors.tds.net/spark/spark-2.4.0/spark-2.4.0-bin-hadoop2.7.tgz
+tar -xf spark-2.4.0-bin-hadoop2.7.tgz
+cd spark-2.4.0-bin-hadoop2.7
+```
+
+If running the count example, download the Alluxio examples jar.
+```bash
+wget https://alluxio-examples.s3.amazonaws.com/spark/alluxio-examples_2.12-1.0.jar
+cp <path_to_alluxio_examples>/alluxio-examples_2.12-1.0.jar jars/
+```
+
+### Short-circuit operations
+
+Short-circuit access enables an Alluxio client in a Spark executor to access the Alluxio
+worker storage on the host machine directly.
+
+If a domain socket location was setup on hosts running the Alluxio worker process at location
```

Comment:
We say what to do if domain socket is set up for Alluxio. Can we also cover what to do if it hasn't been set up? e.g. should they continue to the next section and take a performance hit, or should they set it up? How big of a performance hit?

Reply:
added some text and pointed to the alluxio deployment page for a discussion on domain sockets